x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/parser/SqlParser.java

```diff
@@ -240,26 +261,28 @@ public void syntaxError(Recognizer<?, ?> recognizer, Object offendingSymbol, int
     private static class ParametrizedTokenSource implements TokenSource {
 
         private TokenSource delegate;
-        private Map<Token, SqlTypedParamValue> paramTokens;
-        private int param;
+        private Map<Token, SqlParameter> paramTokens;
+        private int paramIndex;
         private List<SqlTypedParamValue> params;
 
-        ParametrizedTokenSource(TokenSource delegate, Map<Token, SqlTypedParamValue> paramTokens, List<SqlTypedParamValue> params) {
+        ParametrizedTokenSource(TokenSource delegate,
+                                Map<Token, SqlParameter> paramTokens,
+                                List<SqlTypedParamValue> params) {
             this.delegate = delegate;
             this.paramTokens = paramTokens;
             this.params = params;
-            param = 0;
+            paramIndex = 0;
         }
 
         @Override
         public Token nextToken() {
             Token token = delegate.nextToken();
             if (token.getType() == SqlBaseLexer.PARAM) {
-                if (param >= params.size()) {
+                if (paramIndex >= params.size()) {
                     throw new ParsingException("Not enough actual parameters {} ", params.size());
                 }
-                paramTokens.put(token, params.get(param));
-                param++;
+                paramTokens.put(token, new SqlParameter(paramIndex+1, params.get(paramIndex)));
```

Comment:
incorrect formatting `paramIndex + 1`

Reply:
style: will do, again would be great to add it to `checkStyle`